.gitignore

```diff
@@ -52,4 +52,10 @@ ehthumbs.db
 [Tt]humbs.db
 /test-output
 
-hs_err_pid*
\ No newline at end of file
+hs_err_pid*
```

Comment:
Did you get this from TemplateJavaProject? If not, please refer to the .gitignore there and copy/paste lines from that file.